configuring.md

```diff
@@ -113,6 +113,8 @@ Some additional options can be configured globally by setting these system prope
 * **robolectric.dependency.repo.url** — Set the URL of the Maven repository to use for the runtime dependencies (default `https://repo.maven.apache.org/maven2/`).
 * **robolectric.dependency.repo.username** - Username of repository that you defined in **robolectric.dependency.repo.url**.
 * **robolectric.dependency.repo.password** - Password of repository that you defined in **robolectric.dependency.repo.url**.
+* **robolectric.dependency.proxy.host** — Set the host of the proxy to use for the runtime dependencies.
```

Comment:
These two properties are only available from Robolectric 4.10, what about adding version requirements for it?

Reply:
I forgot about that, but now it's change